components/tools/OmeroWeb/omeroweb/webclient/decorators.py

```diff
@@ -191,3 +192,35 @@ def load_settings(self, request, context, conn):
             c_plugins.append({
                 "label": label, "include": include, "plugin_id": plugin_id})
         context['ome']['center_plugins'] = c_plugins
+
+        open_with = settings.OPEN_WITH
```

Comment:
Like @aleksandra-tarkowska has done with #4706, these settings and this logic need to be tested.